{
  "uid" : "b2a30d3381ede197",
  "name" : "GetBalance Async",
  "fullName" : "GetBalance Async",
  "historyId" : "GetBalance Async",
  "time" : {
    "start" : 1636140166100,
    "stop" : 1636140166563,
    "duration" : 463
  },
  "status" : "failed",
  "statusMessage" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04280a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
  "statusTrace" : "   at Nethereum.JsonRpc.Client.ClientBase.HandleRpcError(RpcResponseMessage response, String reqMsg)\n   at Nethereum.JsonRpc.Client.ClientBase.SendInnerRequestAsync[T](RpcRequestMessage reqMsg, String route)\n   at Nethereum.JsonRpc.Client.ClientBase.SendRequestAsync[T](RpcRequest request, String route)\n   at NeonEndToEnd.org.neonlabs.e2e.nethereum.Balance.GetBalance() in /home/runner/work/neon-e2e/neon-e2e/src/main/java/org/neonlabs/e2e/nethereum/Balance.cs:line 14\n   at NeonEndToEnd.Tests.org.neonlabs.e2e.nethereum.ConnectionTest.ShouldGetBalanceAsync() in /home/runner/work/neon-e2e/neon-e2e/src/test/java/org/neonlabs/e2e/nethereum/ConnectionTest.cs:line 38\n   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_0(Object state)\n   at Xunit.Sdk.AsyncTestSyncContext.<>c__DisplayClass7_0.<Post>b__1(Object _) in C:\\Dev\\xunit\\xunit\\src\\xunit.execution\\Sdk\\AsyncTestSyncContext.cs:line 75",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "thread",
    "value" : "14"
  }, {
    "name" : "host",
    "value" : "fv-az28-213"
  }, {
    "name" : "testClass",
    "value" : "NeonEndToEnd.Tests.org.neonlabs.e2e.nethereum.ConnectionTest"
  }, {
    "name" : "testMethod",
    "value" : "GetBalance Async"
  }, {
    "name" : "package",
    "value" : "NeonEndToEnd.Tests.org.neonlabs.e2e.nethereum.ConnectionTest"
  }, {
    "name" : "suite",
    "value" : "Nethereum"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 25,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 7,
        "unknown" : 0,
        "total" : 32
      },
      "items" : [ {
        "uid" : "fdb3080f4cfdc2dd",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/235//#testresult/fdb3080f4cfdc2dd",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067c190>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1636139910123,
          "stop" : 1636139910848,
          "duration" : 725
        }
      }, {
        "uid" : "d8eec592d77b777d",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/234//#testresult/d8eec592d77b777d",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067ca00>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1636139706509,
          "stop" : 1636139707217,
          "duration" : 708
        }
      }, {
        "uid" : "e1ded0f8c60bb6d0",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/233//#testresult/e1ded0f8c60bb6d0",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067caf0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1636139250306,
          "stop" : 1636139251359,
          "duration" : 1053
        }
      }, {
        "uid" : "78c91ebf6b69bdd",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/232//#testresult/78c91ebf6b69bdd",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7b9a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1636137006138,
          "stop" : 1636137006676,
          "duration" : 538
        }
      }, {
        "uid" : "ad7b82569dcc7c78",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/231//#testresult/ad7b82569dcc7c78",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a18371f0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1636109136264,
          "stop" : 1636109136993,
          "duration" : 729
        }
      }, {
        "uid" : "582b7a6c44739f59",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/230//#testresult/582b7a6c44739f59",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067cca0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1636107121898,
          "stop" : 1636107122990,
          "duration" : 1092
        }
      }, {
        "uid" : "d5a1d4c83b343587",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/229//#testresult/d5a1d4c83b343587",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a183cf10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1636023542332,
          "stop" : 1636023542987,
          "duration" : 655
        }
      }, {
        "uid" : "d730cd10bd32f29d",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/228//#testresult/d730cd10bd32f29d",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067cd00>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635834479992,
          "stop" : 1635834480844,
          "duration" : 852
        }
      }, {
        "uid" : "701550adeb72d60",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/227//#testresult/701550adeb72d60",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0433fa0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635629020429,
          "stop" : 1635629021167,
          "duration" : 738
        }
      }, {
        "uid" : "4f9818f892dbb372",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/226//#testresult/4f9818f892dbb372",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a05105e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635626103633,
          "stop" : 1635626104356,
          "duration" : 723
        }
      }, {
        "uid" : "2135e25f12a0caa7",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/225//#testresult/2135e25f12a0caa7",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a18406d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635625980158,
          "stop" : 1635625980985,
          "duration" : 827
        }
      }, {
        "uid" : "5f41c3a913dc522d",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/224//#testresult/5f41c3a913dc522d",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a1840d30>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635625744955,
          "stop" : 1635625745675,
          "duration" : 720
        }
      }, {
        "uid" : "f08c6fed7face339",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/223//#testresult/f08c6fed7face339",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067cdf0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635621402929,
          "stop" : 1635621403640,
          "duration" : 711
        }
      }, {
        "uid" : "10640a028bd34641",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/222//#testresult/10640a028bd34641",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067c5e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635620684750,
          "stop" : 1635620685546,
          "duration" : 796
        }
      }, {
        "uid" : "d0dc51c7db9c8fa9",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/221//#testresult/d0dc51c7db9c8fa9",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a041f400>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635610162390,
          "stop" : 1635610163124,
          "duration" : 734
        }
      }, {
        "uid" : "4b9e3ef6bf24f9df",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/220//#testresult/4b9e3ef6bf24f9df",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be8c760>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635609694288,
          "stop" : 1635609694851,
          "duration" : 563
        }
      }, {
        "uid" : "a637ef91e8fb9ef9",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/219//#testresult/a637ef91e8fb9ef9",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0449130>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635609050688,
          "stop" : 1635609051523,
          "duration" : 835
        }
      }, {
        "uid" : "970f9e63f4053995",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/218//#testresult/970f9e63f4053995",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067c340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635608405588,
          "stop" : 1635608406421,
          "duration" : 833
        }
      }, {
        "uid" : "524a84e1060d22a",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/217//#testresult/524a84e1060d22a",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07c3f10>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635608138962,
          "stop" : 1635608139471,
          "duration" : 509
        }
      }, {
        "uid" : "d60b1adfbc169d3c",
        "reportUrl" : "https://neonlabsorg.github.io/neon-e2e/216//#testresult/d60b1adfbc169d3c",
        "status" : "failed",
        "statusDetails" : "HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07c3430>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')): eth_getBalance",
        "time" : {
          "start" : 1635607123819,
          "stop" : 1635607124555,
          "duration" : 736
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "b2a30d3381ede197.json",
  "parameterValues" : [ ]
}